src/App.tsx

```diff
@@ -3,20 +3,30 @@ import './App.css';
 import Header from './components/Header';
 import {BrowserRouter as Router, Switch, Route} from 'react-router-dom';
 import HospitalSearch from './components/HospitalSearch';
+import HospitalMap from './components/HospitalMap';
+import { render } from '@testing-library/react';
```

Comment:
Don't include testing code in production binaries.

Reply:
Thanks. Not sure why `render` was there in the first place, but it was removed with no problems.